moveit_py/src/moveit/moveit_core/robot_state/robot_state.cpp

```diff
@@ -0,0 +1,507 @@
+           R"(
+           Set all joints to their default positions.
+           The default position is 0, or if that is not within bounds then half way between min and max bound.
+           )")
+
+      .def("set_to_default_values",
+           py::overload_cast<const moveit::core::JointModelGroup*, const std::string&>(
+               &moveit::core::RobotState::setToDefaultValues),
+           py::arg("joint_model_group"), py::arg("name"),
+           R"(
+           Set the joints in group to the position name defined in the SRDF.
+           Args:
+               joint_model_group (:py:class:`moveit_py.core.JointModelGroup`): The joint model group to set the default values for.
+               name (str): The name of a predefined state which is defined in the robot model SRDF.
+           )")
+
+      .def("set_to_default_values",
+           py::overload_cast<std::shared_ptr<moveit::core::RobotState>&, const std::string&, const std::string&>(
+               &moveit_py::bind_robot_state::set_to_default_values),
+           py::arg("joint_model_group_name"), py::arg("name"),
+           R"(
+           Set the joints in group to the position name defined in the SRDF.
+           Args:
+               joint_model_group_name (str): The name of the joint model group to set the default values for.
+               name (str): The name of a predefined state which is defined in the robot model SRDF.
+       )")
+
+      .def("set_to_random_positions", py::overload_cast<>(&moveit::core::RobotState::setToRandomPositions),
+           R"(
+           Set all joints to random positions within the default bounds.
+           )")
+
+      .def("set_to_random_positions",
+           py::overload_cast<const moveit::core::JointModelGroup*>(&moveit::core::RobotState::setToRandomPositions),
+           py::arg("joint_model_group"),
+           R"(
+           Set all joints in the joint model group to random positions within the default bounds.
+           Args:
+               joint_model_group (:py:class:`moveit_py.core.JointModelGroup`): The joint model group to set the random values for.
+           )")
+
+      .def("clear_attached_bodies", py::overload_cast<>(&moveit::core::RobotState::clearAttachedBodies),
+           R"(
+      	   Clear all attached bodies.
+
+      	   We only allow for attaching of objects via the PlanningScene instance. This method allows any attached objects that are associated to this RobotState instance to be removed.
+	   )")
+
+      .def("update", &moveit_py::bind_robot_state::update, py::arg("force") = false, py::arg("type") = "all",
+           R"(
+             Update state transforms.
+
+             Args:
+                 force (bool):
+             category (str): specifies the category to update. All indicates updating all transforms while "links_only"
+             and "collisions_only" ensure that only links or collision transforms are updated. )");
```

Comment:
indentation is weird

Reply:
[b026ee5](https://github.com/ros-planning/moveit2/pull/1546/commits/b026ee57f3a9af2f9e41c124255b7e577b07b685)